windtunnel/windtunnel_outputs.py

```diff
@@ -0,0 +1,153 @@
+
+    Attributes:
+        simulation_path (str): Path to simulation output directory.
+        time_steps (int): Number of time steps in the simulation.
+        domain_mesh (pv.PolyData): Domain mesh from the simulation.
+        object_mesh (pv.PolyData): Object mesh from the simulation.
+    """
+
+    def __init__(self, simulation_path: str):
+        """
+        Initializes WindTunnelOutputs with the simulation directory.
+
+        Args:
+            simulation_path (str): Path to simulation output directory.
+        """
+        self.simulation_path = simulation_path
+        self.time_steps = self._get_num_time_steps()
+        self.domain_mesh, self.object_mesh = self._get_output_meshes()
+
+    def get_force_coefficients(self):
+        """
+        Loads force coefficients from the simulation output.
+
+        Returns:
+            dict: Force coefficients including 'Moment', 'Drag', 'Lift',
+            'Front Lift', and 'Rear Lift'.
+        """
+        force_coefficients_path = os.path.join(self.simulation_path,
+                                               "postProcessing", "forceCoeffs1",
+                                               "0", "forceCoeffs.dat")
+        coefficients = np.loadtxt(force_coefficients_path)[-1]
+        coefficients_labels = [
+            "Moment", "Drag", "Lift", "Front Lift", "Rear Lift"
+        ]
+        coefficients_dict = dict(
+            zip(coefficients_labels, map(float, coefficients[1:])))
+
+        return coefficients_dict
+
+    def get_streamlines(self,
+                        n_points=100,
+                        initial_step_length=1,
+                        source_radius=1.1,
+                        source_center=(0, 0, 0),
+                        streamline_radius=0.005):
+        """
+        Generates streamlines from the domain mesh.
+
+        Args:
+            n_points (int, optional): Number of seed points. Defaults to 100.
+            initial_step_length (float, optional): Initial step length.
+                                                   Defaults to 1.
+            source_radius (float, optional): Radius of seeding sphere.
+                                             Defaults to 1.1.
+            source_center (tuple, optional): Center of seeding sphere.
+                                             Defaults to (0, 0, 0).
+            streamline_radius (float, optional): Radius of streamlines.
+                                                 Defaults to 0.005.
+
+        Returns:
+            pv.PolyData: Streamlines as tube-shaped meshes.
+        """
+        streamlines_mesh = self.domain_mesh.streamlines(
+            max_time=self.time_steps,
+            n_points=n_points,
+            initial_step_length=initial_step_length,
+            source_radius=source_radius,
+            source_center=source_center)
+        streamlines = streamlines_mesh.tube(radius=streamline_radius)
+        return streamlines
+
+    def get_pressure_field(self):
```

Comment:
maybe make the name more explicit, like get_openfoam_object_mesh()

Reply:
```suggestion
    def get_openfoam_object_mesh(self):
```